Applications/Core TLP.md

```diff
@@ -0,0 +1,69 @@
+
+## History & Metrics
+
+Created by Ryan Dahl, now the largest programming ecosystem in the world.
+
+## Scope
+
+The Core TLP will have sole responsibility and discretion over the node.js project in the following areas:
+
+* Setting release dates
+* Release quality standards
+* Technical direction
+* Governance process and practices.
+* Contribution process and practices.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators and Working Groups.
+* Node.js build and CI infrastructure.
+
+## Governance
+
+https://github.com/nodejs/node/blob/master/GOVERNANCE.md
+
+## Contributions
+
+https://github.com/nodejs/node/blob/master/CONTRIBUTING.md
+
+## Tools
+
+* GitHub `nodejs` org.
+* Uberconference & Soundcloud
+* Google Hangouts & Youtube
+
+## IP
+
+All relevant IP is already managed by the Node.js Foundation.
+
+## TC Members
+
+* **Ben Noordhuis** &lt;info@bnoordhuis.nl&gt; ([@bnoordhuis](https://github.com/bnoordhuis))
+* **Bert Belder** &lt;bertbelder@gmail.com&gt; ([@piscisaureus](https://github.com/piscisaureus))
+* **Fedor Indutny** &lt;fedor.indutny@gmail.com&gt; ([@indutny](https://github.com/indutny))
+* **Trevor Norris** &lt;trev.norris@gmail.com&gt; ([@trevnorris](https://github.com/trevnorris))
+* **Chris Dickinson** &lt;christopher.s.dickinson@gmail.com&gt; ([@chrisdickinson](https://github.com/chrisdickinson))
+* **Rod Vagg** &lt;rod@vagg.org&gt; ([@rvagg](https://github.com/rvagg))
+* **Jeremiah Senkpiel** &lt;fishrock123@rocketmail.com&gt; ([@fishrock123](https://github.com/fishrock123))
+* **Colin Ihrig** &lt;cjihrig@gmail.com&gt; ([@cjihrig](https://github.com/cjihrig))
+* **Alexis Campailla** &lt;orangemocha@nodejs.org&gt; ([@orangemocha](https://github.com/orangemocha))
+* **Julien Gilli** &lt;jgilli@nodejs.org&gt; ([@misterdjules](https://github.com/misterdjules))
+* **James M Snell** &lt;jasnell@gmail.com&gt; ([@jasnell](https://github.com/jasnell))
+* **Shigeki Ohtsu** &lt;ohtsu@iij.ad.jp&gt; ([@shigeki](https://github.com/shigeki))
+* **Brian White** &lt;mscdex@mscdex.net&gt; ([@mscdex](https://github.com/mscdex))
+
+## Working Groups
+
+* [Website](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#website)
+* [Streams](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#streams)
+* [Build](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#build)
+* [Tracing](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#tracing)
+* [Roadmap](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#roadmap)
+* [Docker](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#docker)
+* [Addon API](https://github.com/nodejs/node/blob/master/WORKING_GROUPS.md#addon-api)
+
```

Comment:
i18n groups are much wider than core so the plan is to keep them as a top level WG. This makes it a bit easier to engage with them about localization and other opportunities related to other TLPs.